popup.js

```diff
@@ -0,0 +1,423 @@
+    workTools: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  },
+  {
+    title: 'Uber Navigation',
+    workDetails: ['FACEBOOK', 'Back End Dev', '2015'],
+    image: './images/Portfolio 4.png',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    details:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    details2:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+
+    workTools: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+
+    liveIcon: './images/live-Icon.svg',
+    liveLink: 'https://kayembajonathan.github.io/mydestop_pro/',
+
+    sourceLink: 'https://kayembajonathan.github.io/mydestop_pro/',
+    sourceIcon: './images/bluegithub.svg',
+  },
+];
+
+// ------------------------- Projects section --------------------//
+
+const project = document.querySelector('.grid-container');
+project.innerHTML = `<div class="card">
```

Comment:
- [Optional] Consider looping through the project array to update the project section, this will save you from repeating the same card over and over for each project.
```javascript
portfolioData.forEach((project) => {
  project.innerHTML += `<div class="card">
    <h2>{project.title}</h2>
      ....
    </div>
`
});
```

Reply:
@Kamba56 i disagre with that

Reply:
At the moment, the code is not respecting the DRY principle. 
Imagine that we have 1,000 cards that we need to display. The code written would be huge. And on top of that, maybe we need to make some changes to the way we display the data on the UI. The workload required to change each and every card cannot be calculated.

If we use a loop, we only have very few lines of code to build and maintain.

For now, let's keep this requirement Optional, but for the Capstone project, it would be better to implement a smarter way to display the data on the UI.

Also, I noticed that the cards are not taking the data from the correct object. 
At the moment, each and every card is taking data from multiple objects.
![image](https://user-images.githubusercontent.com/83596135/224136778-e783efcc-9bf8-45b3-a927-5138a9709937.png)

I highly advise rendering the UI details for a card from the corresponding object in the array (as an optional step since @Kamba56 did not ask for it).